Makefile

```diff
@@ -115,7 +115,8 @@ check-slides: build  ## check the markdown-formatted links in slides
 
 check-yaml: ## lint yaml files
 	( $(ACTIVATE_ENV) && \
-	  find . -path "**/*.yaml" | xargs -L 1 -I '{}' sh -c "yamllint {}" \
+	  find . -name "*.yaml" | xargs -L 1 -I '{}' sh -c "yamllint {}" \
```

Comment:
I thinkg we should also use single quotes for `*.yaml` to prevent shell expansion, i.e. `find . -name '*.yaml' | xargs ...`

Reply:
I don't believe expansion happens when double or single quoted but I could be wrong? I was only under the impression that variables are expanded, not file paths.

Reply:
ok, will do @nsoranzo :)

Reply:
Ah, @erasche is right, sometimes Bash is so confusing!